Client/mods/deathmatch/logic/CClientEntity.cpp

```diff
@@ -303,6 +303,20 @@ CLuaArgument* CClientEntity::GetCustomData(const char* szName, bool bInheritData
     return NULL;
 }
 
+CLuaArguments* CClientEntity::GetAllCustomData(CLuaArguments* table)
+{
+    assert(table);
+
+    // Grab it and return a pointer to the variable
+    for (auto it = m_pCustomData->IterBegin(); it != m_pCustomData->IterEnd(); it++)
+    {
+        table->PushString(it->first.c_str());                // key
```

Comment:
is `.c_str()` actually required? I have a hunch it will work without.

Reply:
We can remove it and see

Reply:
Yeah, It's working fine

Reply:
select the function and then do ADvanced -> Format Selection :)

Reply:
Done, we need just the return type I guess now.